code-challenges.test.js

```diff
@@ -24,11 +34,35 @@
 
 // b) Create the function that makes the test pass.
 
+// create a function named "fibonacciSeq"
+// create a param of array
+// loop to iterate up to the number that we input
+// add the output to the previous number that was outputted 
+
+const fibonacciSeq = (array) => {
+  let num = []
+  num[0] = 1
+  num[1] = 1
+  for (let i = 2; i < array; i++){
+    num.push(num[i - 2] + num[i - 1])
+  }
+  return num
+}
+
+ console.log(fibonacciSeq(8));
+ 
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("sortedOdd", () => {
+  it("takes in an array and returns a new array of only odd numbers sorted from least to greatest", () => {
+    expect(sortedOdd(fullArr1)).toEqual([-9, 7, 9, 199])
+    expect(sortedOdd(fullArr2)).toEqual([-823, 7, 23])
+  })
+})
```

Comment:
Looks great.  Try to keep your test variables inside your it method within your test.  While having them outside of the test will work, best practice is keeping them in the scope that they are being used in.